NoLoSay/NoLoApp
# Adds script to update version

## What type of PR is this? (check all applicable)

- [ ] Other (please precise) : 

## Description

Adds a version script to update all the versions of the app

## Related Tickets & Documents

- Related Issue [NOLO-170](https://nolosay.atlassian.net/jira/software/c/projects/NOLO/boards/1?modal=detail&selectedIssue=NOLO-170)

## QA Instructions, Screenshots, Recordings

Try to run `bin/version` and see if every file is updated (package.json, app.json, ios/NoLoSay/Info.plist, android/app/build.gradle

## Added/updated tests?

- [x] No, and this is why: Not in the scope

## [optional] What gif best describes this PR or how it makes you feel?
![](https://github.com/NoLoSay/NoLoApp/assets/25590592/6fdf1a2d-04bd-4894-b0e3-bd8e61b8d2f5)


[NOLO-170]: https://nolosay.atlassian.net/browse/NOLO-170?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
